harness/determined/common/api/authentication.py

```diff
@@ -81,7 +82,9 @@ def _init_session(
             token = util.get_container_user_token()
 
         if token is not None:
-            return Session(session_user, token)
+            return Session(
+                session_user, token, _get_current_user_id(self.master_address, token, cert)
```

Comment:
seems like a sort of unfortunate new round trip on every start up. though at least when we do a login we save the trip (nice). only way I see to get around it would also be to store the user id in the token store (and inject it for task containers somehow), which is kind of a pain.

Reply:
I agree - why don't we store the user id in the token store?
then I think we'd only need to fetch it for users who are upgrading from the previous version without requiring them to log in again, if that.

Reply:
updating to use token store instead